core/jazz_ui/src/app/pages/service-detail/service-detail.component.html

```diff
@@ -19,7 +19,7 @@ <h1 class="page-hdr bold-title">{{service.name}}</h1>
                 <span *ngIf="service.runtime"> - {{service.runtime}}</span>
             </span>
             </div>
-            <div class="image-btn-div">
+            <div class="image-btn-div flex-end">
```

Comment:
Why is this changed needed in the service detail? What issue is this addressing?

Reply:
CSS class was added in service-detail as a precaution and to have code in sync between env-detail and service-detail.